src/pages/Leaderboard.tsx

```diff
@@ -1,10 +1,37 @@
 import '../styles.css'
+import { useEffect, useState} from "react"
 
 function Leaderboard() {
 
+    const [users, setUsers] = useState([])
+    const [loading, setLoading] = useState(false)
+    useEffect(() => {
+        setLoading(true)
+        fetch("https://ylikellotus.lajp.fi/")
+        .then(response => response.json())
+        .then(json => setUsers(json))
+        .finally(() => {
+            setLoading(false)
+        })
+    }, [])
+    
+
     return (
         <div>
-            <p>Leaderboard tähän?</p>
+            <table>
+                <tr>
+                    <td>Nimi</td>
+                    <td>Aika</td>
+                    <td>Kilta</td>
```

Comment:
We want to get the localized texts (Nimi, Aika, Kilta) from Localization.tsx, so that we can have english and finnish versions depending on selected lang